app/src/main/java/com/android/feedme/ui/LandingScreen.kt

```diff
@@ -184,3 +191,66 @@ fun RecipeList(padding: PaddingValues, recipes: List<Recipe>) {
     }
   }
 }
+
+/**
+ * Function that displays our search bar as well as the filters icon
+ *
+ * @param searchText : the default text displayed in the search bar
+ * @param onSearchTextChanged : if we want to apply any changes to the text we type in the search
+ *   bar
+ * @param onFilterClicked : the function that will allow us to go through filters by clicking on the
+ *   icon
+ */
+@Composable
+fun SearchBar(
```

Comment:
Shouldn't this be a https://developer.android.com/reference/com/google/android/material/search/SearchBar instead?